tests/TraceContext/W3CTestService/symfony-setup

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Install Symfony
+curl -sS https://get.symfony.com/cli/installer | bash
```

Comment:
Always worth being aware this will install the most recent version of symfony, which may break in the future.  We'll cross that bridge when we get there.

Reply:
Good catch! I've pinned the version to the current latest (5.2) for now and we can update it as we go along.